libs/vue-plugin/src/builders/browser/builder.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  BuilderContext,
+  BuilderOutput,
+  createBuilder
+} from '@angular-devkit/architect';
+import { join, normalize } from '@angular-devkit/core';
+import { from, Observable, of } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+import { BrowserBuilderSchema } from './schema';
+import { getProjectRoot } from '../../utils';
+import {
+  modifyCachePaths,
+  modifyEntryPoint,
+  modifyIndexHtmlPath,
+  modifyTsConfigPaths
+} from '../../webpack';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const Service = require('@vue/cli-service/lib/Service');
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { resolvePkg } = require('@vue/cli-shared-utils/lib/pkg');
+
+export function runBuilder(
+  options: BrowserBuilderSchema,
+  context: BuilderContext
+): Observable<BuilderOutput> {
+  if (!options.outputPath) {
+    context.logger.error('Invalid options, "outputPath" is required.');
+    return of({ success: false });
+  }
+
+  const inlineOptions = {
+    chainWebpack: config => {
+      modifyIndexHtmlPath(config, options, context);
+      modifyEntryPoint(config, options, context);
+      modifyTsConfigPaths(config, options, context);
+      modifyCachePaths(config, context);
+    }
+  };
+
+  return from(getProjectRoot(context)).pipe(
+    map(
+      projectRoot =>
+        new Service(projectRoot, {
+          pkg: resolvePkg(context.workspaceRoot),
+          inlineOptions
+        })
+    ),
+    switchMap(service =>
+      from(
+        service.run(
+          'build',
+          {
+            mode: options.mode,
+            dest: join(normalize(context.workspaceRoot), options.outputPath),
```

Comment:
Tested on my Windows machine and we have the same path problem. Let's replace all occurrences of `join(normalize(path1), path2)` with nodes default path join `path.join(path1, path2)`

Reply:
I fixed this playing the Angular DevKit game. For now.

Reply:
All commands are working on my Windows machine now 👍